schema.sql

```diff
@@ -3,3 +3,11 @@ create table animals (id integer, name varchar(20), data_of_birth date,escape_at
 
 alter table animals 
 add species varchar(255);
+
+alter table animals alter column id set default nextval('animals');
```

Comment:
- [ ] Please from the project requirement, You should :point_down: 
  - Create a table named `owners` with the following columns:
    - `id`: integer (set it as autoincremented PRIMARY KEY)
    - `full_name`: string
    - `age`: integer
  - Create a table named `species` with the following columns:
    - `id`: integer (set it as autoincremented PRIMARY KEY)
    - `name`: string

Currently, you dont have any `SQL` in this schema.sql indicating `owners` and `species` tables.